modules/shortcodes/flickr.php

```diff
@@ -148,39 +138,31 @@ function flickr_shortcode_handler( $atts ) {
 
 	if ( 'video' === $showing ) {
 
-		if ( ! is_numeric( $src ) && ! preg_match( '~^(https?:)?//([\da-z\-]+\.)*?((static)?flickr\.com|flic\.kr)/.*~i', $src ) ) {
-			return '';
-		}
```

Comment:
I would leave this check here. It allows us to bail early for funky domains, such as numeric URL